system-parachains/asset-hubs/asset-hub-polkadot/tests/tests.rs

```diff
@@ -786,9 +780,9 @@ pub mod remove_when_updated_to_stable2409 {
 		<WeightToFee as frame_support::weights::WeightToFee>::Balance: From<u128> + Into<u128>,
 		SovereignAccountOf: ConvertLocation<AccountIdOf<Runtime>>,
 		<Runtime as pallet_assets::Config<ForeignAssetsPalletInstance>>::AssetId:
-			From<xcm::v4::Location> + Into<xcm::v4::Location>,
+			From<Location> + Into<Location>,
 		<Runtime as pallet_assets::Config<ForeignAssetsPalletInstance>>::AssetIdParameter:
-			From<xcm::v4::Location> + Into<xcm::v4::Location>,
+			From<Location> + Into<Location>,
```

Comment:
@claravanstaden I think all the changes bellow are part of the temporary hack:
```
#[allow(clippy::too_many_arguments)]
pub mod remove_when_updated_to_stable2409 {
```
all the relevant test-cases here should be already in the `asset-test-utils` (which use xcmv5).

I think this is the best time/place to remove `remove_when_updated_to_stable2409`, so it will simplify other stuff here

Reply:
Done in https://github.com/polkadot-fellows/runtimes/pull/753/commits/de3a2e269fc1d421d126d824c082b1ae979a5ff2#diff-1370964dffc6108b28fa548324d4b0c6768415b685fc89c663c37dd05352754c.

Reply:
nice!!!
![image](https://github.com/user-attachments/assets/4a83cb3e-7529-4a2f-bfdb-803396af6893)